src/esn.calendar.libs/app/services/sidebar-visibility.service.js

```diff
@@ -0,0 +1,21 @@
+angular.module('esn.calendar.libs')
+  .service('calendarSidebarService', calendarSidebarService);
+
+function calendarSidebarService($rootScope) {
+  let isSidebarVisible = true;
+
+  function getSidebarVisibility() {
+    return isSidebarVisible;
+  }
+
+  function setSidebarVisibility(sidebarVisibility) {
+    isSidebarVisible = sidebarVisibility;
+
+    $rootScope.$broadcast('change-sidebar-visibility');
```

Comment:
just a small comment, the `change-sidebar-visibility` should be declared in an angular constant.